addons/sale/views/sale_portal_templates.xml

```diff
@@ -594,4 +596,35 @@
             <t t-call="account.document_tax_totals"/>
         </table>
     </template>
+
+    <template id="sale_order_amount_already_paid">
+        <t t-if="sale_order.state in ['draft', 'sent', 'sale']">
+            <t t-if="sale_order.state in ['draft' ,'sent']">
+                <t t-set="type_of_order" t-value="'Quotation'"/>
+            </t>
+            <t t-else="">
+                <t t-set="type_of_order" t-value="'Sales Order'"/>
+            </t>
```

Comment:
I think this is too much. Customers will see if this is a quotation or a sale order by the title